src/prop/minisat/minisat.h

```diff
@@ -22,6 +22,14 @@
 #include "util/statistics_registry.h"
 
 namespace cvc5 {
+
+template <class Solver>
```

Comment:
Are these going to be deleted eventually as well?  Or why are they currently needed?

Reply:
I had to add them here because they were previously defined in the deleted sat proof files. We currently need them because of the existence of BVMinisat. Since we want to remove that eventually, these could go at that time as well.

Reply:
Ok cool, that's what I expected.